demo/json_component/run.py

```diff
@@ -1,6 +0,0 @@
-import gradio as gr 
```

Comment:
This demo is used in the docs so we can't delete it

<img width="833" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/91466f51-0da0-4176-b2aa-76f3e3ed6d71">

Reply:
Oh missed it, sorry.
Revert renaming.